engines/sci/parser/vocabulary.h

```diff
@@ -47,6 +47,7 @@ enum {
 	VOCAB_RESOURCE_SELECTORS = 997,
 
 	VOCAB_RESOURCE_SCI0_MAIN_VOCAB = 0,
+	VOCAB_RESOURCE_SCI0_LOC_VOCAB = 1,
```

Comment:
This should be renamed and commented to make it clear that this is a third party extra and not a part of "real" SCI

Reply:
Added comment and renamed